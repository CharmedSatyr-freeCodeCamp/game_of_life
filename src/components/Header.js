import React from 'react';

const Header = () => (
  <header>
    <h1>John Conway's Game of Life</h1>
    <small>a zero player automaton</small>
  </header>
);

export default Header;
