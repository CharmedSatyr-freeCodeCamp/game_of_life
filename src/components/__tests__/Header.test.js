import React from 'react';
import { shallow } from 'enzyme';

import Header from '../Header';

describe('<Header /> component', () => {
  it('renders without crashing', () => {
    const header = shallow(<Header />);
    expect(header.exists()).toBeTruthy();
  });
});
