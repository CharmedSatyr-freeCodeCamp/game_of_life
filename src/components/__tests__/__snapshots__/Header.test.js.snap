// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> component renders correctly 1`] = `
<header>
  <h1>
    John Conway's Game of Life
  </h1>
  <h3>
    a zero player automaton
  </h3>
</header>
`;
